operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/ExportOperation.java

```diff
@@ -61,7 +61,7 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
             String logicalId, String versionId, Parameters parameters, FHIRResourceHelpers resourceHelper)
             throws FHIROperationException {
         common.checkEnabled();
-        common.checkAllowed(operationContext);
+        common.checkAllowed(operationContext, true);
```

Comment:
shouldn't this be false?
```suggestion
        common.checkAllowed(operationContext, false);
```